### PR TITLE
Add sample notebook for image classification training with list format

### PR DESCRIPTION
**Pull request by xzhou33**

### CODE REVIEW

**Comment by xzhou33** [comment]
@gurumurthys  fix review comments

**Comment by bkowshik** [comment]
I am not able to 👀 the notebook on Github. Locally running `jupyter notebook` see this error:

`Unreadable Notebook: /Users/test/repos/xzhou33/amazon-sagemaker-examples/introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-lst-format.ipynb NotJSONError('Notebook does not appear to be JSON: \'{\\n "cells": [\\n {\\n "cell_type": "m...',)`

**Comment by xzhou33** [comment]
@djarpin  Fixed the notebook format

File: introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-lst-format.ipynb [hunk 1]
```diff
@@ -0,0 +1,733 @@
...
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "sage.delete_endpoint(EndpointName=endpoint_name)"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
```
Comment (djarpin) [inline introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-lst-format.ipynb]: Can you add the below copyright statement to the notebook metadata?

`"notice": "Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.  Licensed under the Apache License, Version 2.0 (the \"License\"). You may not use this file except in compliance with the License. A copy of the License is located at http://aws.amazon.com/apache2.0/ or in the \"license\" file accompanying this file. This file is distributed on an \"AS IS\" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.",`

File: introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-lst-format.ipynb [hunk 2]
```diff
@@ -0,0 +1,733 @@
...
+   "outputs": [],
+   "source": [
+    "# get the status of the endpoint\n",
+    "response = sagemaker.describe_endpoint(EndpointName=endpoint_name)\n",
+    "status = response['EndpointStatus']\n",
+    "print('EndpointStatus = {}'.format(status))\n",
+    "\n",
+    "\n",
+    "# wait until the status has changed\n",
+    "sagemaker.get_waiter('Endpoint_Created').wait(EndpointName=endpoint_name)\n",
```
Comment (djarpin) [inline introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-lst-format.ipynb]: Can you add the waiter logic from [here](https://github.com/awslabs/amazon-sagemaker-examples/blob/master/sagemaker-python-sdk/1P_kmeans_lowlevel/kmeans_mnist_lowlevel.ipynb) to this notebook?

File: introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-lst-format.ipynb [hunk 3]
```diff
@@ -0,0 +1,733 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import boto3\n",
+    "runtime = boto3.Session().client(service_name='sagemaker-runtime') "
```
Comment (djarpin) [inline introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-lst-format.ipynb]: I think this has to be "runtime.sagemaker" to work outside of a SageMaker Notebook Instance.

File: introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-lst-format.ipynb [hunk 4]
```diff
@@ -0,0 +1,733 @@
...
+    "\n",
+    "sage = boto3.Session().client(service_name='sagemaker') \n",
+    "\n",
+    "timestamp = time.strftime('-%Y-%m-%d-%H-%M-%S', time.gmtime())\n",
+    "model_name=\"image-classification-model\" + timestamp\n",
+    "print(model_name)\n",
+    "info = sage.describe_training_job(TrainingJobName=job_name)\n",
+    "model_data = info['ModelArtifacts']['S3ModelArtifacts']\n",
+    "print(model_data)\n",
+    "containers = {'us-west-2': '433757028032.dkr.ecr.us-west-2.amazonaws.com/image-classification:latest',\n",
```
[the remark this answers is not in the document]
Comment (xzhou33) [reply introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-lst-format.ipynb]: Removed.

File: introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-lst-format.ipynb (hunk 1 above)
Comment (xzhou33) [reply introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-lst-format.ipynb]: Added to metadata.

File: introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-lst-format.ipynb (hunk 2 above)
Comment (xzhou33) [reply introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-lst-format.ipynb]: Added.

File: introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-lst-format.ipynb (hunk 3 above)
Comment (xzhou33) [reply introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-lst-format.ipynb]: Changed.

**Comment by xzhou33** [comment]
@djarpin  fix review comments.

**Pull request merged**